app/namespace.go

```diff
@@ -1176,257 +1176,254 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 		},
 	}
 
-	// ----------- Commands for private review feature, only available when feature flag turns on -----------
-	// Export Command
-	if IsFeatureEnabled(ExportFeatureFlag) {
```

Comment:
The only change is remove 

`if IsFeatureEnabled(ExportFeatureFlag) {`

Rest are formatting.